### DISCUSSION TITLE
HELP. Senior newbie cannot get program to automatically uninstall bloatware my new computer

### DISCUSSION THREAD

**Discussion by ReddFraser** [Klocman/Bulk-Crap-Uninstaller#497] [Q&A]
Good day. I am a senior very near useless on computers. A computer site recommended having this uninstaller to automatically remove bloatware on my new computer, and old dead files etc from my older computer. I have spent over an hour but cannot find any button that will start the automatic uninstall. The program shows a long page of assorted items, and I have absolutely no idea which ones are bloatware or how to remove them. Did I misunderstand that this should happen automatically? Could someone, who knows what they're doing, please send me a very detailed step by step way to automatically get rid of the bloatware? I would say please pretend I am an 8 year old, but they would most likely know how to do it already....thanks for your time, Redd     PS: could you please let me know via reddfraser@hotmail.com, as I probably will not be able to find this box again...

**Comment by Klocman** [comment]
You have to select what you want to remove manually, BCU does not remove anything that you don't tell it to. If you don't know what is and isn't bloatware, you might want to look for a different tool that's designed for beginners instead.
